app/services/preserved_object_handler.rb

```diff
@@ -261,19 +263,42 @@ def update_pc_invalid_moab
       pres_copy = PreservedCopy.find_by!(preserved_object: pres_object, endpoint: endpoint) if pres_object
       # FIXME: what if there is more than one associated pres_copy?
       update_status(pres_copy, PreservedCopy::INVALID_MOAB_STATUS)
-      update_pc_audit_timestamps(pres_copy, true, false)
+      update_pc_audit_timestamps(pres_copy, false)
       update_db_object(pres_copy)
     end
     handler_results.remove_db_updated_results unless transaction_ok
   end
 
-  def update_pc_unexpected_version(pres_copy, pres_object, new_status, ran_moab_validation)
+  # given a PreservedCopy instance and whether the caller found the expected version of it on disk, this will perform
+  # other validations of what's on disk, and will update the status accordingly
+  def set_status_as_seen_on_disk(pres_copy, found_expected_version)
+    # TODO: when we implement C2M, we should add a check here to look for the object on disk, and set PreservedCopy::ONLINE_MOAB_NOT_FOUND_STATUS
+    # if needed.  or maybe the caller will have done that already, since it needs to try reading the moab to obtain incoming_version?
```

Comment:
this assumes C2M will be in pohandler.  That is ... unclear at this time.   Should this comment stay?

Reply:
good point, i'd been assuming C2M would leverage POH methods that called this new method, but that seems less clear after the discussions this afternoon.  happy to remove this comment or make it appropriately vague.

Reply:
I dunno what to do;  confusion reigns in what some what call my "mind"